tests/e2e/instrumentation-java/00-install-instrumentation.yaml

```diff
@@ -3,11 +3,26 @@ kind: Instrumentation
 metadata:
   name: java
 spec:
-  exporter:
```

Comment:
any reason why this is removed? 

Is your intention to override exporter config by explicitly setting exporter env vars? If this is the case I would prefer to keep the exporter config.